docs/platform/what-is-tizen/versions/tizen-8-0-m2.md

```diff
@@ -0,0 +1,134 @@
+# Tizen 8.0
+
+The second milestone (M2) release of Tizen 8.0 was announced in October 2023. Tizen is an open-source operating system (OS) maintained by Samsung, and it has been developed and optimized for smart devices since 2012.
+
+The main progress of Tizen 8.0 includes the work that has been done to expand the ecosystem of home appliances. Tizen 8.0 is also updated and optimized to unlock even more experiences for our developers. These new enhancements include a multi-modal interaction framework, 3D UI, machine learning, build infra, and more.
+
+## Multi-modal Interaction Framework
```

Comment:
```suggestion
## Multi-modal interaction framework
```

Reply:
@safir-srbd Hello, Thank you for your review. But I think this formatting is used in the sub title level. So if we change these 2 formatting, we have to change a lot. Please let me know if you still want to change these formatting.

Reply:
@jsngyoon In that case, no worries. We keep it as is for now.

Reply:
@safir-srbd OK. Thanks.